src/main/java/io/swagger/service/MediaService.java

```diff
@@ -1,40 +1,66 @@
 package io.swagger.service;
 
 import io.swagger.entity.media.MediaInfo;
+import io.swagger.model.media.SearchData;
+import io.swagger.model.media.SearchResult;
+import io.swagger.model.media.TitleData;
 import io.swagger.repository.MediaRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Optional;
 
 // The service will fetch data from imdb API if data does not already
 // exist in local database
+@Service
 public class MediaService {
 
     @Autowired
     private MediaRepository mediaRepository;
 
+    @Autowired
+    private IMDBService imdbService;
+
+    private Map<String, TitleData> mediaCache = new HashMap<>();
+    private Map<String, SearchData> searchCache = new HashMap<>();
+
+    public TitleData GetMediaById(String id) {
+        //Optional<TitleData> mediaInfo = mediaRepository.findById(id);
```

Comment:
Storing to the database didn't work quite like I expected. There are issues with storing lists (which would be stored as a string if we serialize/deserialize) and the plot column needs to be larger than 255 (which I am not sure why my schema.sql script didn't work to increase that size).

For now, I have the cache as the HashMaps above. This will work as designed/intended.... I was hoping to get the database portion working.